target/linux/ath79/generic/base-files/etc/hotplug.d/ieee80211/10_fix_wifi_mac

```diff
@@ -65,4 +65,7 @@ case "$board" in
 		[ "$PHYNBR" -eq 1 ] && \
 			mtd_get_mac_ascii u-boot-env ethaddr > /sys${DEVPATH}/macaddress
 		;;
+	zyxel,nwa1121-ni)
+		mtd_get_mac_text mib0 0x4b > /sys${DEVPATH}/macaddress
```

Comment:
If you overwrite the WiFi MAC address here, we would you then use the original WiFi MAC address for ethernet?

Reply:
`eth0mac` from `mib0` (at `0x30`) is indeed the same as the binary one located in `art` at `0x1002`, which is used for ethernet, while `wifi0mac` is located at `0x4b` in `mib0`.
Since there is only a dt binding for the binary format, only the ethernet mac can be set directly from the dts node (though it is indeed a little awkward that this is stored in the position that would usually hold wifi calibration data).

Reply:
Please preserve this information in a MAC address overview table in your commit message:
https://openwrt.org/docs/guide-developer/device-support-policies#mac_addresses